plugins/PostProcessingPlugin/scripts/UsePrevProbeMeasure.py

```diff
@@ -0,0 +1,46 @@
+# Cura PostProcessingPlugin
+# Author:   Amanda de Castilho
+# Date:     January 5,2019
+
+# Description:  This plugin overrides probing command and inserts code to ensure
+#               previous probe measurements are loaded and bed leveling enabled
+#               (searches for G29 and replaces it with M501 & M420 S1)
+#               *** Assumes G29 is in the start code, will do nothing if it isn't ***
+
+from ..Script import Script
+
+class UsePrevProbeMeasure(Script):
+    def __init__(self):
+        super().__init__()
+
+    def getSettingDataString(self):
+        return """{
+            "name": "Use Previous Probe Measurements",
+            "key": "UsePrevProbeMeasure",
```

Comment:
Use "UsePreviousProbeMeasurements" instead
```suggestion
            "key": "UsePreviousProbeMeasurements",
```